templates/configure.yaml

```diff
@@ -22,10 +22,11 @@ spec:
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     spec:
       containers:
+        {{ include "rhtap.trustification.configure" . | indent 8 }}
```

Comment:
I try to keep list sorted alphabetically so they are easier to parse whenever possible.

Reply:
Sure!